src/side-navigation/side-navigation.scss

```diff
@@ -1,205 +1,144 @@
-  overflow: hidden;
   transition: max-width $iui-speed-fast ease-out;
   height: 100%;
-
   @include themed {
-    background-color: t(iui-color-background-1);
-    border: 1px solid t(iui-color-background-5);
+    background-color: t(iui-color-background-3);
+    border-right: 1px solid t(iui-color-background-5);
   }
 
-  > .iui-expand-container {
-    height: $iui-l;
-    cursor: pointer;
-    display: inline-flex;
-    justify-content: center;
-    align-items: center;
-
-    @include themed {
-      background-color: t(iui-color-background-1);
-    }
-
-    > .iui-icon {
-      width: $iui-icons-small;
-      height: $iui-icons-small;
-
-      @include themed {
-        fill: t(iui-icons-color);
-      }
-    }
-
-    &:hover > .iui-icon {
-      @include themed {
-        fill: t(iui-icons-color-primary);
-      }
-    }
-
-    &:focus {
-      outline: none;
-    }
-
-    &:focus-visible {
-      @include themed {
-        box-shadow: inset 0 0 $iui-xs
-          rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-3));
-      }
-    }
-  }
-
-  > .iui-sidenav-content {
+  > .iui-content {
     display: flex;
     flex-direction: row;
-    height: calc(100% - #{$iui-l});
+    height: calc(100% - #{$iui-line-height});
 
-    > .iui-sidenav-items-container {
+    > .iui-content-container {
       display: flex;
       flex-direction: column;
+      width: 100%;
 
-      > .iui-sidenav-items {
+      > .iui-top {
         flex-grow: 1;
         display: flex;
         flex-direction: column;
-      }
 
-      > .iui-sidenav-items > .iui-sidenav-item-container,
-      > .iui-bottom-container > .iui-sidenav-item-container {
-        margin-bottom: -1px;
-        @include iui-sidenav-item-container;
+        > .iui-button:not(:last-of-type) {
+          border-bottom: none;
+        }
+      }
+      
+      > .iui-bottom > .iui-button {
+        border-bottom: none;
       }
     }
   }
 
   &.iui-collapsed {
-    min-width: $iui-l * 2;
-    max-width: $iui-l * 2;
+    min-width: $iui-icons-large * 3;
+    max-width: $iui-icons-large * 3;
   }
 
   &.iui-expanded,
-  > .iui-sidenav-content > .iui-sidenav-items-container {
+  > .iui-content > .iui-content-container {
     max-width: $iui-3xl * 3;
   }
 
-  &.iui-expanded > .iui-expand-container > .iui-icon {
+  &.iui-expanded > .iui-expand > .iui-icon {
     transform: rotate(180deg);
-  }
-}
-
-@mixin iui-sidenav-item-container {
-  display: flex;
-  flex-direction: row;
-  user-select: none;
-  height: $iui-l * 2;
-  cursor: pointer;
-  align-items: center;
-  transition: box-shadow $iui-speed-fast ease;
-  text-decoration: none;
-  box-sizing: border-box;
-  border-right: none;
-  border-left: none;
-
-  @include themed {
-    border-top: 1px solid t(iui-color-background-5);
-    border-bottom: 1px solid t(iui-color-background-5);
-    background-color: t(iui-color-background-1);
-  }
-
-  &:not(:last-child) {
-    border-bottom: none;
+    // NOT WORKING
+    animation: 5s ease roll;
```

Comment:
Try changing to `transition: transform $iui-speed-fast ease-out`.  Might need to also add `fill` here 
![image](https://user-images.githubusercontent.com/36186912/118098678-dd2fb380-b3dc-11eb-99ca-0e80351ef582.png)
Result:
![sidenav_transition](https://user-images.githubusercontent.com/36186912/118098701-e28cfe00-b3dc-11eb-903a-43fa09cf1d1c.gif)

Reply:
Nice, thank you!